src/RefractAPI.cc

```diff
@@ -39,6 +41,18 @@ namespace drafter {
             }
         }
 
+        refract::ArrayElement* CreateArrayElement(refract::IElement* value, bool rFull = false)
```

Comment:
When I said adding a parameter, I meant something like the following function signature

```cpp
    template <typename T>
    refract::ArrayElement* CreateArrayElement(const T& content, bool rFull, bool isContentElement)
```

But that's not important. Important thing is duplication of code. Can we have something like the following?

```cpp
    namespace {
        refract::ArrayElement* CreateArrayElement(refract::IElement* value, bool rFull = false)
        {
            refract::ArrayElement* array = new refract::ArrayElement;

            if (rFull) {
                value->renderType(refract::IElement::rFull);
            }

            array->push_back(value);
            return array;
        }
    }

    template <typename T>
    refract::ArrayElement* CreateArrayElement(const T& content, bool rFull)
    {
        refract::IElement* value = refract::IElement::Create(content);
        return CreateArrayElement(value, rFull);
    }
```